macrosAutoCatsTests/src/test/resources/test-cases/todo/ambiguesTraitImplementation.failure

```diff
@@ -0,0 +1,15 @@
+import com.softwaremill.macwire.autocats._
+
+trait T
+case class A(t: T) extends T
+class B() extends T
+
+object Test {
+  val theA = autowire[A](new B())
```

Comment:
why is this ambiguous? B is the only instance of type T that we have?

Reply:
yeah, fixed it: https://github.com/softwaremill/macwire/pull/188/files#diff-b9ff332c41e5e2363b8dc27fe12be0dc1da96d6d6834b8017e293ddbab9e7355